src/enforester.js

```diff
@@ -1852,7 +1851,7 @@ export class Enforester {
       return stx.addScope(introducedScope, this.context.bindings, ALL_PHASES, { flip: true });
     });
 
-    return result;
+    return result.concat(ctx[Symbol.for('enforester')].rest);
```

Comment:
Referencing a global symbol feels a little dirty, but I'm struggling to find a better way.